paasta_tools/paasta_remote_run_2.py

```diff
@@ -0,0 +1,101 @@
+import json
+from time import sleep
+
+from kubernetes.client.exceptions import ApiException
+
+from paasta_tools.kubernetes.application.controller_wrappers import (
+    get_application_wrapper,
+)
+from paasta_tools.kubernetes_tools import KubeClient
+from paasta_tools.kubernetes_tools import load_kubernetes_service_config_no_cache
+from paasta_tools.utils import DEFAULT_SOA_DIR
+
+
+def remote_run_start(service, instance, user, cluster, interactive, recreate):
+    # TODO Overriding the kube client config for now as the api has limited permissions
+    kube_client = KubeClient(config_file="/etc/kubernetes/admin.conf")
+
+    # TODO hardcoded for now
+    is_eks = False
+
+    # Load the service deployment settings
+    deployment = load_kubernetes_service_config_no_cache(
+        service, instance, cluster, DEFAULT_SOA_DIR
+    )
+    namespace = deployment.get_namespace()
+
+    # Set to interactive mode
+    if interactive:
+        deployment.config_dict["cmd"] = "sleep 604800"  # One week
+
+    # Create the app with a new name
+    formatted_application = deployment.format_kubernetes_app()
+    formatted_application.metadata.name += f"-remote-run-{user}"
```

Comment:
there's the risk of the pod name being truncated, so we'll definitely also want to indicators of this being a remote-run in the pod's labels